deployments/data102/hubploy.yaml

```diff
@@ -1,7 +1,7 @@
 images:
   images:
     - name: us-central1-docker.pkg.dev/ucb-datahub-2018/user-images/primary-user-image
-      path: ../datahub/images/default
+      path: ../image/
```

Comment:
I think if you use images.image_name, you don't need images.path. data100 is also an example. See `git grep image_name` for other deployment examples.